src/CMakeLists.txt

```diff
@@ -2,8 +2,10 @@
 if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
     find_library(coreFoundation CoreFoundation)
     set(PLATFORM_TARGET_LIBS ${coreFoundation})
-elseif(MINGW)
+elseif(MINGW) # Should this be more general?
```

Comment:
I think `imm32.lib` is only needed for `ImmGetContext` or some other Windows related thing, i.e. if you build with MinGW while using `win32.cpp` source, but I don't remember exactly.